### DISCUSSION TITLE
Repository Activity

### DISCUSSION THREAD

**Discussion by JStuve** [microsoft/vsts-extension-retrospectives#563] [General]
Hello! I'm looking to get this repository a little more activity than the previous 3 months. Historically there have been several PRs merged a month with frequent releases. There are a lot of teams that utilize this tool and is constantly a top installed extension. Would love to keep contributing knowing that we can work together to fix bugs and get new features out.

**Comment by polatengin** [comment]
hey @JStuve ,

I noticed your contributions, and finally got a chance to review and merge them 👍

First of all, let me thank you for your contributions.

And just like you noticed, I was able to be more _active_ on this repo before than now.

I'd _love_ to have the community back me up when I can't do all the things.

If you think you can frequently contribute, let's have a discussion about how I can make your life easier 😄

**Reply by JStuve** [reply]
Awesome, thanks for your time as always, let me know the best place to contact you for further discussion.
